app/src/main/res/values/strings.xml

```diff
@@ -155,4 +155,5 @@
     <string name="no_edit">No edits detected</string>
     <string name="copy_message_fetch">Copying message to clipboard</string>
     <string name="copy_failed">Copying to clipboard failed</string>
+    <string name="message_not_found">Message not found</string>
```

Comment:
Technically this message is right, but you should change this string as the user will kinda annoyed as they would have long press on the message itself! 
maybe something like Something went wrong, try again